### PR TITLE
Basic server binary (sentry, logging, no-op GRPC service, etc)

### PR DESCRIPTION
**Pull request by eranrund**

### CODE REVIEW

**Comment by eranrund** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/2" target="_blank">Graphite</a> 
    * **PR #3** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/3" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/3?utm_source=stack-comment).

File: server/src/server.rs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        );
+
+        let health_service =
+            mc_util_grpc::HealthService::new(None, self.logger.clone()).into_service();
+
+        let client_service = ClientService::new(self.logger.clone()).into_service();
+
+        let grpc_env = Arc::new(
+            grpcio::EnvBuilder::new()
+                .name_prefix("Ingest-RPC".to_string())
```
Comment (cbeck88) [inline server/src/server.rs:63]:
```suggestion
                .name_prefix("Deqs-RPC".to_string())
```

File: server/src/server.rs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            .register_service(health_service)
+            .register_service(client_service);
+
+        let mut server =
+            server_builder.build_using_uri(&self.client_listen_uri, self.logger.clone())?;
+        server.start();
+
+        log::info!(
+            self.logger,
+            "Ingest GRPC API listening on {}",
```
Comment (cbeck88) [inline server/src/server.rs:77]:
```suggestion
            "Ingest Deqs API listening on {}",
```

**Review by cbeck88 — APPROVED** [review]
looks good

**Pull request merged**